README.md

```diff
@@ -0,0 +1,83 @@
+# Oppiabot
+
+<p align="center">
+    <img src="images/oppiabot-display-image.png">
+</p>
+
+Oppiabot is a GitHub app built with [probot](https://github.com/probot/probot). It acts as a helper for the Oppia code respository to maintain the development workflow. It is hosted on [Heroku](https://www.heroku.com/).
+
+The bot currently performs the following functions:
+
+1. Checks whether a contributor creating a pull request has signed the CLA. It adds a label `Needs CLA` accordingly.
+
+2. Checks merge conflict and build status of a pull request and notifies the author as well as the maintainer-on-duty regarding the same.
+
+3. Closes stale pull requests.
+
+
+## Getting started
+
+Please refer to the following instructions to setup Oppiabot for the first time on your machine:
+
+1. Create a new, empty folder called `opensource/` within your home folder. Navigate to it (`cd opensource`), then [fork and clone](https://help.github.com/articles/fork-a-repo/) the Oppiabot repo. This will create a new folder named `opensource/oppiabot`. Navigate to `opensource/oppiabot/`.
+
+2. Install [Node.Js](https://nodejs.org/) and [NPM](https://www.npmjs.com/) on your machine using the following commands:
+
+  ```bash
+    sudo apt-get install nodejs
+    sudo apt-get install npm
+  ```
+
+3. Setup probot and other dependencies by running the following command:
+  ```bash
+    npm install
+  ```
+
+4. The Oppiabot uses environment variables. These are configured in the server settings. To deploy the bot locally, create a `.env` file and copy the contents of `.env.example` to it.
+
+5. Go to [smee.io](https://smee.io/) and click **Start a new channel**. Set `WEBHOOK_PROXY_URL` in `.env` to the URL that you are redirected to.
+
+6. [Create a new GitHub App](https://github.com/settings/apps/new) with:
+    * **Webhook URL**: Use your `WEBHOOK_PROXY_URL` from the previous step.
+    * **Webhook Secret**: `development`
+    * **Permissions & events** is located lower down the page and will depend on what data you want the bot to have access to. Currently, the bot requires read and write access to issues and pull requests and read-only access to the file in the path `.github/stale.yml`.  
+    If, for example, you only enable issue events, you will not be able to listen on pull request webhooks with the bot.
+
+7. Download the private key. It will be a `.pem` file. Move it to the root directory. As long as it's in the root directory, it will be automatically detected regardless of the filename.
```

Comment:
Root directory of the project or user account?

Reply:
Done, thanks!